### ISSUE TITLE
about pre-training loss

### ISSUE THREAD

**Issue by biburger** [KeNiu042/Diffusion-ReID#3]
Thanks for your contribution!
 I am very interested in how you pretrained your ResNet50 and ViT backbone, since you didn't mention which loss function was used in your work. Specifically, in Table 10, do you mean using the "diff-person" dataset and the "soldier" method?
